server/index.js

```diff
@@ -19,15 +19,23 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-const path = require('path'),
-  Koa = require('koa'),
-  bodyparser = require('koa-bodyparser'),
-  send = require('koa-send'),
-  staticRoot = path.join(__dirname, '../dist'),
-  app = new Koa(),
-  router = require('./routes');
+const path = require('path');
+const Koa = require('koa');
```

Comment:
i'll update. casing ultimately has no difference in javascript but its good to keep to convention.

Reply:
Looking at the code more closely, the code actually calls new on Koa. this would indicate that this is a class (or constructor). Typically in javascript we notate classes vs functions by using first capital letter. So i think in this particular case the casing is correct.

You can look at this convention here:
https://stackoverflow.com/questions/1564398/javascript-method-naming-lowercase-vs-uppercase/1564489#:~:text=5%20Answers&text=A%20popular%20convention%20in%20Javascript,mistakenly%20called%20%22classes%22).&text=Anything%20that's%20not%20a%20constructor%20usually%20starts%20with%20lowercase%20and%20is%20camelCased.